docs/components/icons/Example.vue

```diff
@@ -3,35 +3,23 @@ import decamelize from "decamelize";
 import * as icons from "@warp-ds/icons/vue";
 import * as reactIcons from "@warp-ds/icons/react";
 import { wModal } from "@warp-ds/vue";
-import { ref } from "vue";
+import { computed, ref } from "vue";
 
-// When an existing icon is deprecated, add { old: "OldIconName", new: "NewIconName" } to the deprecatedIcons list.
+// When an existing icon is deprecated, add { old: "IconOldName16", new: "IconNewName16" } to the deprecatedIcons list.
 // This will display a deprecation message with a suggested replacement.
-const deprecatedIcons = []
+const deprecatedIcons = [{old:'IconAds16', new:'IconAirCon16'},{old:'IconHonkDark42'}];
```

Comment:
Is IconAds16 and IconHonkDark42 deprecated, or was it a display of how we could present deprecated icons (which I like btw)?
![Screenshot 2024-06-10 at 14 47 14](https://github.com/warp-ds/tech-docs/assets/41303231/ccc5abb0-1f26-44ac-bad6-5cb192f7c767)

Reply:
Ah. Crap. That's just for testing... 😅 I'll remove it. 👍